server/app/controllers/TaskSolver.scala

```diff
@@ -67,5 +100,9 @@ object TaskSolver {
 
   def sbt(command: String): Try[Boolean] = Try(Seq("sbt", command).! == 0)
 
-  lazy val sbtInstalled = sbt("sbtVersion").isSuccess // no exception, so sbt is in the PATH
+  // no exception, so sbt is in the PATH
+  lazy val sbtInstalled = sbt("--version").isSuccess
```

Comment:
dorlov@dmytro13:~$ sbt --version
Java HotSpot(TM) 64-Bit Server VM warning: ignoring option MaxPermSize=384m; support was removed in 8.0
sbt launcher version 0.13.7